src/Api/Client.php

```diff
@@ -318,7 +318,7 @@ protected function _arrayToXml(array $array, SimpleXMLElement $xml, $parentEl =
             if (is_array($value)) {
                 $this->_arrayToXml($value, $this->_isAssocArray($value) ? $xml->addChild($el) : $xml, $el);
             } else {
-                $xml->addChild($el, $value);
+                $xml->$el = $value;
```

Comment:
I suppose there must be escape function like this
$value = htmlspecialchars($value, ENT_QUOTES | ENT_XML1);

Reply:
There is an escape function, but API endpoint does not unescape this value: it gets stored in the database as-is and leaves account with the 'qwerty&amp;' password instead of the 'qwerty&'